tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+# role tasks
+- block:
+  - name: Install cockpit
+    yum:
+      name: "{{ cockpit_version }}"
+      state: present
+
+  - include_tasks: start_service.yml
```

Comment:
You don't need an include for a single task. The `become` and `become_user` can go right on the task itself.